packages/runtime/container-runtime/src/summary/runningSummarizer.ts

```diff
@@ -200,7 +201,7 @@ export class RunningSummarizer implements IDisposable {
 		const maxAckWaitTime = Math.min(this.configuration.maxAckWaitTime, maxSummarizeAckWaitTime);
 
 		this.pendingAckTimer = new PromiseTimer(maxAckWaitTime, () => {
-			// Note: summarizeCount (from ChildLogger definition) may be 0,
+			// Note: summarizeCount (from createChildLogger definition) may be 0,
```

Comment:
Nit: the original comment doesn't make sense to me, is the update necessary?

Reply:
```suggestion
			// Note: summarizeCount (from ChildLogger definition) may be 0,
```